tests/visualization_tests/test_param_importances.py

```diff
@@ -1,9 +1,8 @@
+from __future__ import annotations
+
 from io import BytesIO
 from typing import Any
 from typing import Callable
-from typing import List
-from typing import Optional
-from typing import Tuple
```

Comment:
I think it can be deleted, too.
`Tuple` was used only in Line 193, and it has already been properly modified.

Reply:
Sorry, it is already resolved. This comment was my mistake.